### ISSUE TITLE
documentation style

### ISSUE THREAD

**Issue by albertz** [PerBothner/DomTerm#53]
This is just a suggestion.
Currently, the homepage looks extremely outdated, like something from the 90ies. When I saw it first, I closed it again as I thought this project is probably abadoned since a long time. Only accidently I found later this GitHub repo and was very confused that it is still alive, based on the homepage I have seen earlier. This is not a good impression, esp not for such a project, which is really actually something extremely modern. (Of course, some other people maybe would go the other extreme, declare this as a **next-gen terminal emulator** in bold fonts, with some very fancy style. This is also not what I'm suggestion here.)

Some options:

* Maybe changing the background color, the font, and maybe slightly the CSS already fixes this. (I'm really not a web designer, so unfortunately I cannot really help there.)

* Maybe just use [readthedocs.io](http://readthedocs.io) instead? This integrates nicely into GitHub. You can have the documentation directly in the repository, and every time you push, it will automatically update the docs.

* Or [GitHub pages](https://pages.github.com/), and some of the existing default style files (which all look very simplistic and modern).

**Comment by PerBothner** [comment]
I'm not sure what makes it "outdated".  I think the basic model of a sidebar plus main panel is very functional and elegant.  It seems to me it's exactly the same as ReadTheDocs. It works with pretty much any screen or font size, and scrolling works correctly, which many "modern" pages fail to do, I'm old-school and I like simple elegant fast-loading easy-to-navigate web-pages without a lot of bling.  I also like pages that don't require a fancy server.

I do think there is too much text on the front page.  Probably the "Why another terminal emulator" section should be moved.

_"Maybe changing the background color, the font, and maybe slightly the CSS already fixes this"_

I'm certainly open to suggestions here.  Maybe I can tweak the style to look more like ReadThedocs or GitHub pages, if you think that would help.

Nothing wrong with ReadThedocs or GitHub pages.  However, I want to use my own domain name.  I also find that in terms of actually writing technical documentation, good old texinfo is still pretty good, and certainly better than Markdown.  I haven't found anything clearly better.  (Trust me, I've looked: see the use of asciidoctor for help files, which I'd be happier with if the project was more active.) The default html output from texinfo does suck, but DomTerm doesn't use it as-is.

**Comment by albertz** [comment]
>> Maybe changing the background color, the font, and maybe slightly the CSS already fixes this

> I'm certainly open to suggestions here. Maybe I can tweak the style to look more like ReadThedocs or GitHub pages, if you think that would help.

Yes, I think this already would help a lot. Many GitHub pages are conceptionaly very similar to the DomTerm website, in that they are often very simple as well. However, the style makes it look much more modern.

**Comment by PerBothner** [comment]
Please take a look at: http://per.bothner.com/tmp/domterm/
I changed the lightblue used for headers to a even lighter blue, removed the thin border around the headers, and got rid of the uppercasing.  I also added a 1em padding around the main body.

**Comment by albertz** [comment]
Btw, also see the [website of Hyper](https://hyper.is/), a very similar project, which looks much more polished/modern (and simpler).

**Comment by PerBothner** [comment]
I don't really like the Hyper home-page: Personally I dislike white-on-black, finding it harder to read. Also, the home page is too big, with option and API documentation that should be in a different page. I don't find anything modern about it, at least in a positive sense.

The dynmic-gif demo is nice, but the home-page should be just that, plus the "project goals", plus links.

**Comment by PerBothner** [comment]
I editing the home page to make it less verbose, which is desirable in itself.

I made some more styling tweaks. Just now I played a bit with the styling of links. I do think it is helpful to keep the traditional blue underline for links, but I made the underline lighter, and changed the background color to match the links in the (now-lighter) navigation bar.

I disagree with some of your preferences. I think the navigation column should have a different background color than the main contents, for example. I like a distinct background color for titles, though I agree a more subtle (lighter) color works better.

By "The text spacing should be increased" do you mean padding? If so, where? I increased the padding around the main text area.

You'd like the logo to be improved.  So would I, but it isn't easy - I'm not a graphics artist. I want it to look like a terminal, and I like that it incorporates the text "DomTerm".  I think it works well enough that tweaking it is not a priority.

I am not going to make screenshots on Mac, or using a Mac-like theme. That doesn't mean I'll reject contributed Mac screenshots, but I am enough of a GNU/Linux supporter to support that first.  I don't equate Mac with modern.

**Comment by PerBothner** [comment]
Thanks for your input.  I made the logo a bit smaller, and I also fixed how "hover" works over it.

The `DT> _` text is "Courier 10 pitch".  The logo is created with Inkscape, so changing the fonts is easy.  Probably a monospace font without serifs would be better, but I haven't made up my mind.  It might also make sense to replace the underline with one of the Unicode block elements, either "full block" `█ ` or "Lower one eight block `▁`, but both may be a bit "heavy".

I'm not sure the web-sides you linked to all "look better and more modern". They're fine, but I like the functionality and look of the DomTerm site. (I could probably do with improving my other web-sites.)  I may get back to style issues later, but at this point I think it makes sense to get used to what I have before I spend more time on it.

Again, thanks for your thoughtful and helpful advice.
